google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/StreamWriterV2.java

```diff
@@ -59,6 +59,11 @@
    */
   private final String streamName;
 
+  /*
+   * The proto schema of rows to write.
+   */
+  private final ProtoSchema writerSchema;
```

Comment:
Could this be Descriptor? The user will need to call the conversion method (which is one of our library) in order to get to here.

Reply:
I think it is similar either way. And we already agree on this in the last discussion. I would like to avoid go back and get everybody agree on a new thing given it is similar either way.

Reply:
This is not schema compact check. I just want to bring it up. I am fine keeping this interface more open and leave options to users.